Makefile

```diff
@@ -0,0 +1,62 @@
+#TODO: ADD PHONY install-cudnn
```

Comment:
can we do this?

Reply:
I have removed the target that I originally made to install system dependencies ( including `install-cudnn`).

I install the system dependencies in the instructions for building the targets that needs them and I made sure that the the scripts for installing check if it already ran (or to be precise, if the libraries are already there). 

The PHONY targets are now `setup clean pyinstaller clean-pyinstaller`  + _fictional_ targets (= aliases that makes the makefile easier to test)